src/main/java/io/appium/java_client/pagefactory/README.md

```diff
@@ -0,0 +1,160 @@
+# Standard Selectors
+
+## AndroidFindBy / iOSXCUITFindBy / WindowsFindBy
+
+# Advanced Selectors
+
+## iOS's String Predicates
+
+You can specify a [predicate](https://developer.apple.com/documentation/foundation/nspredicate)
+in your Class Chain to limit the number of matched items. There's
+[a detailed guide to that](https://appium.io/docs/en/writing-running-appium/ios/ios-predicate/index.html)
+on the Appium Docs website with some Appium-specific considerations.
+
+## iOS's Class Chain Queries
+
+Our XCUiTest integration has full support for the 'Class Chain' concept. This
+can do much of what XPath does...but faster. Note that many Class Chains leverage
+String Predicates too.
+
+### String Predicates in Class Chains
+
+There's a special array-style syntax for defining predicates in a Class Chain:
+
+```
+// Start with [
+// Followed by `
+// Followed by some text
+// Followed by `
+// End with ]
+[`label != 'a'`]
+[`isWDVisible == 1`]
+[`value < 0`]
+```
+
+#### Searching Descendents
+
+If you replace the backticks (`) around a predicate with dollar signs ($), then
```

Comment:
backticks are also used for GitHub markup. Please escape them properly

Reply:
If I knew how I would....the only thing I can think of is an html entity like &#96;. Is that allowed?

Reply:
I found a github issue https://github.com/github/markup/issues/363 with some sort of guidance. I'll try using double backticks around it, like so: `` ` ``